src/main/java/org/vaadin/example/bookstore/ui/about/AboutView.java

```diff
@@ -8,19 +9,20 @@
 import com.vaadin.flow.server.Version;
 import org.vaadin.example.bookstore.ui.MainLayout;
 
+import java.util.ResourceBundle;
+
 @Route(value = "About", layout = MainLayout.class)
 @PageTitle("About")
 public class AboutView extends HorizontalLayout {
 
-    public static final String VIEW_NAME = "About";
+    private transient ResourceBundle resourceBundle = ResourceBundle.getBundle("MockDataWords", UI.getCurrent().getLocale());
 
     public AboutView() {
         add(VaadinIcon.INFO_CIRCLE.create());
-        add(new Span(" This application is using Vaadin version "
+        add(new Span(" " + resourceBundle.getString("application_sign") + " "
```

Comment:
In Farsi it is should be written like this:
```
add(new Span(“این برنامه از وادین نسخه ”
        + Version.getFullVersion() + ” استفاده می کند.“));
```

Reply:
Updated.